validator/src/main.rs

```diff
@@ -885,6 +885,12 @@ pub fn main() {
                        slots behind the highest snapshot available for \
                        download from other validators"),
         )
+        .arg(
+            // NOP arg for forward compatibility with ISS on-by-default in v1.10
+            Arg::with_name("no_incremental_snapshots")
+                .long("no-incremental-snapshots")
+                .hidden(true)
```

Comment:
Looks good. I think you could also add an [`.conflicts_with()`](https://docs.rs/clap/2.34.0/clap/struct.Arg.html#method.conflicts_with) here to proactively deal with issues where one script/envar sets `--incremental-snapshots` and another one sets `--no-incremental-snapshots`.

Reply:
good call!